data/pid_data.py

```diff
@@ -5390,6 +5391,33 @@
   'set_sub_device_range': 1,
   'value': 0x1050},
 
+ # MANUFACTURER_URL
+ {'get_request': {'items': []},
+  'get_response': {'items': [{'name': 'url',
+                              'min_size': 2,
+                              'type': 'string'}]},
+  'get_sub_device_range': 2,
+  'name': 'MANUFACTURER_URL',
+  'value': 0x00d1},
```

Comment:
should be 0x00d0

Reply:
Looks like I reviewed an old state and this was fixed in the meantime. Thus, Approval without any remarks :)

Reply:
I wonder if that was based on when I requested review and it linking to that not HEAD of the PR?

I've even added some testing to catch that issue in future:
https://github.com/OpenLightingProject/rdm-app/pull/315/commits/45dd9f952007044da3f0c5a705397f1c9656600a